modules/serverless-workflow-tag-revisions.adoc

```diff
@@ -0,0 +1,46 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="tag-revisions_{context}"]
+= Tag revisions 
```

Comment:
This module doesn't really fall into one of the module types. I would consider breaking it up so that assigning and unassigning tags are each their own procedure module.